src/services/users/create-new-user/create-new-user.js

```diff
@@ -0,0 +1,77 @@
+const { 
```

Comment:
the folder was not too required, i think the files could just be in the `users/` folder if there are no other files

but if we want to stick with this, the files inside the `create-new-user` should now be  `index.js` and `index.test.js`